sdk/core/src/main/java/com/klaviyo/core/lifecycle/NoOpLifecycleCallbacks.kt

```diff
@@ -0,0 +1,21 @@
+package com.klaviyo.core.lifecycle
+
+import android.app.Activity
+import android.app.Application
+import android.os.Bundle
+
+/**
+ * A no-op implementation of ActivityLifecycleCallbacks
+ * to temporarily replace the public property Klaviyo.lifecycleCallbacks
+ * and prevent duplicate registration of the KlaviyoLifecycleMonitor
+ * until next major release when we can make the breaking change to remove that public property
+ */
+object NoOpLifecycleCallbacks : Application.ActivityLifecycleCallbacks {
```

Comment:
So this would also be removed in next major release once we no longer need to worry about duplicate lifecycle registration?

Reply:
Correct, I just can't remove it in a minor release by semantic versioning rules